Tools/autotest/sim_vehicle.py

```diff
@@ -1137,10 +1205,42 @@ def generate_frame_help():
         sys.exit(1)
 
     start_vehicle(vehicle_binary,
-                  find_autotest_dir(),
                   cmd_opts,
                   frame_infos,
-                  loc=location)
+                  spawns=spawns)
+
+if cmd_opts.delay_start:
```

Comment:
Duplicate from line 1245?

Reply:
I figured there should be a delay between starting scrimmage and mavproxy as well as between ardupilot and scrimmage.